docs/content/preview/yugabyte-cloud/cloud-secure-clusters/cloud-users.md

```diff
@@ -56,10 +60,22 @@ Although not a superuser, `yb_superuser` includes sufficient privileges to perfo
 
 - Has the following role options: `INHERIT`, `CREATEROLE`, `CREATEDB`, and `BYPASSRLS`.
 
-- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, and [yb_extension](#yb-extension).
+- Member of the following roles: `pg_read_all_stats`, `pg_signal_backend`, `yb_db_admin`, and [yb_extension](#yb-extension).
 
 `yb_superuser` is the highest privileged role you have access to in YugabyteDB Managed. You can't delete, change the passwords, or login using the `postgres` or `yugabyte` superuser roles.
 
+### yb_db_admin
+
+The `yb_db_admin` role is specific to YugabyteDB Managed clusters and allows non-superuser roles to perform a number of privileged operations. A user granted this role can do the following:
+
+- alter and drop schemas it does not own
+- alter and drop tables it does not own
+- alter and drop indexes it does not own
+- create tablespaces on tables and indexes it does not own, and drop tablespaces it does not own
+- alter roles it does not own
```

Comment:
`yb_db_admin` can be thought of a role that passes on superuser privileges. The "it does not own" part places undue emphasis on being able to merely run commands on resources it does not own. It would be better to reword this to more clearly convey the idea that `yb_db_admin` allows users to perform privileged operations much like a superuser (just as a superuser would), of which running operations on objects it does not own being one.

FYI, regarding altering roles, yb_db_admin can only alter the BYPASSRLS attribute of a role. Other things such as giving a role superuser attribute does not work as this would lead to security issues.